examples/cms-starter/README.md

```diff
@@ -0,0 +1,19 @@
+# Framer Plugin Template
+
+This is a template for using the Framer Plugin API in a TypeScript project.
+
+## Quickstart
+
+First, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
+# or
+pnpm dev
+# or
+bun dev
+```
+
+Learn more: https://www.framer.com/developers/plugins/introduction
```

Comment:
Nit: This is @huntercaron's wheelhouse, but, just in case, I'd maybe simplify or even eliminate the first paragraph, as we probably won't be maintaining a non-TypeScript version of this, and the rest is mostly a rehash of the title. And add a direct link to how to open the plugin, to reduce friction just a tiny bit.

````suggestion
# Framer Plugin Template

Run the development server:

```bash
npm run dev
# or
yarn dev
# or
pnpm dev
# or
bun dev
```

And [open in Framer](https://www.framer.com/developers/plugins/quick-start#opening-in-framer).

Learn more: https://www.framer.com/developers/plugins/introduction
````